admin-auth/src/test/kotlin/com/epam/drill/admin/auth/CoffeineCacheServiceTest.kt

```diff
@@ -54,4 +54,10 @@ class CoffeineCacheServiceTest {
             assertNotNull(principal)
         }
 
+    @Test
+    fun `given principle with value null, then cache must not populate`(): Unit = runBlocking {
+        cacheService.getFromCacheOrPutIfAbsent("custom-key-1") { null }
```

Comment:
Add assert that the return value is null.

Reply:
added